### PR TITLE
Remove PAPI v1 CI [BT-80]

### PR DESCRIPTION
**Pull request by mcovarr**
Removing the actual PAPI v1 code is the subject of a [separate issue](https://broadworkbench.atlassian.net/browse/BT-84).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -56,27 +56,20 @@ env:
...
-      BUILD_TYPE=centaurPapiUpgradeNewWorkflowsPapiV2alpha1
-      BUILD_MYSQL=5.7
+    # This is not currently running any tests, see BT-81.
+    # - >-
+    #  BUILD_TYPE=centaurPapiUpgradePapiV2alpha1
+    #  BUILD_MYSQL=5.7
+    # This is not currently running any tests, see BT-81.
+    # - >-
+    #  BUILD_TYPE=centaurPapiUpgradeNewWorkflowsPapiV2alpha1
+    #  BUILD_MYSQL=5.7
```
Comment (kshakir) [inline .travis.yml:66]: Revert, and use the same "syntax" as BA-6547: https://github.com/broadinstitute/cromwell/blob/54/src/ci/bin/testConformanceTesk.sh

Or, update testConformanceTesk.sh and .travis.yml to use this new format.

I just don't think jobs should be removed N different ways. I also prefer the other style of printing a message as it leaves a reminder of missing tests.

File: .travis.yml [hunk 2]
```diff
@@ -56,27 +56,20 @@ env:
...
     - >-
       BUILD_TYPE=centaurLocal
       BUILD_MYSQL=5.7
     - >-
       BUILD_TYPE=centaurLocal
       BUILD_POSTGRESQL=11.3
-    - >-
-      BUILD_TYPE=centaurPapiV1
-      BUILD_MYSQL=5.7
```
Comment (kshakir) [inline .travis.yml:79]: As of 6e977e6bd57ea80e3eef86c98228524151ce8061, this shell script is still in the commit.

**Review by kshakir — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (mcovarr) [reply .travis.yml:79]: oops

**Review by kshakir — APPROVED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -56,12 +56,6 @@ env:
     - >-
       BUILD_TYPE=centaurHoricromtalEngineUpgradePapiV2alpha1
       BUILD_MARIADB=10.3
-    - >-
-      BUILD_TYPE=centaurPapiUpgradePapiV1
-      BUILD_MYSQL=5.7
-    - >-
-      BUILD_TYPE=centaurPapiUpgradeNewWorkflowsPapiV1
-      BUILD_MYSQL=5.7
```
Comment (breilly2) [inline .travis.yml:64]: I think this means we can also delete lines 272-277 from `test.inc.sh` (https://github.com/broadinstitute/cromwell/blob/develop/src/ci/bin/test.inc.sh#L272) as well as `papi_v1_v2alpha1_upgrade_application.conf` (https://github.com/broadinstitute/cromwell/blob/develop/src/ci/resources/papi_v1_v2alpha1_upgrade_application.conf).

Comment (breilly2) [reply .travis.yml:64]: Or maybe only some of that, if we're keeping `testCentaurPapiUpgradeNewWorkflowsPapiV1.sh`?

File: src/ci/bin/test_papi.inc.sh [hunk 4]
```diff
@@ -100,19 +100,7 @@ cromwell::private::papi::setup_papi_gcr() {
...
     CROMWELL_BUILD_PAPI_AUTH_MODE="service-account"
-
-    # Delete all usages of CROMWELL_BUILD_PAPI_JSON_FILE from there if you delete this block!
-    if [[ "${CROMWELL_BUILD_TYPE}" == "centaurPapiV1" ]]; then
-        # Downgrade to an older service account on Papi V1 until refresh_token_no_auth_bucket.test is
-        # migrated/fixed/added for the Papi V2 credential below
-        CROMWELL_BUILD_PAPI_JSON_FILE="${CROMWELL_BUILD_RESOURCES_DIRECTORY}/cromwell-service-account.json"
-    else
-        # This service account does not have billing permission, and therefore cannot be used for requester pays
-        CROMWELL_BUILD_PAPI_JSON_FILE="${CROMWELL_BUILD_RESOURCES_DIRECTORY}/cromwell-centaur-service-account.json"
```
Comment (grsterin) [inline src/ci/bin/test_papi.inc.sh:111]: Shouldn't this one be preserved?

**Review by kshakir — APPROVED** [review]

File: src/ci/bin/test_papi.inc.sh (hunk 4 above)
Comment (mcovarr) [reply src/ci/bin/test_papi.inc.sh:111]: The variable is completely gone now and the literal value is used only [here](https://github.com/broadinstitute/cromwell/pull/6150/files/bf718d4dfcda91c76f07e0e4fe8cb614bbacc5ee#diff-342d95134a3c6c37cf687387f71731e7650273286f0e67fbca9c6cf6d081c64bR28)

**Review by grsterin — APPROVED** [review]

**Pull request merged**